.github/workflows/link_and_spell_check.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+    - main
+
+name: "Check spelling, linting and links"
+
+jobs:
+  check-markdown:
+    runs-on: ubuntu-latest
+    name: "Check spelling, linting and links"
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        wget https://github.com/qir-alliance/.github/archive/refs/heads/main.zip -O clone.zip &&
+        unzip clone.zip && mv .github-main out && rm clone.zip &&
+        mv out/workflow-templates/utils/ utils && rm -r out
+      shell: bash
+    - run: |
+        Get-ChildItem (Join-Path "utils" "problem-matchers") `
+          | ForEach-Object {
+            Write-Host "Adding matcher $_...";
+            Write-Host "::add-matcher::$_";
+          }
+      shell: pwsh
+    - name: markdownlint-cli
+      # cspell:words nosborn
```

Comment:
What does this comment mean?

Reply:
That's where we get this from: https://github.com/nosborn/github-action-markdown-cli